cycledash/genotypes.py

```diff
@@ -108,9 +110,9 @@ def get(vcf_id, query, with_stats=True, truth_vcf_id=None):
         q = _add_filters(q, g, query.get('filters'))
         q = _add_orderings(q, g, query.get('sortBy'))
         q = _add_paging(q, g, query.get('limit'), query.get('page'))
-        q = q.order_by(asc(func.length(g.c.contig)),
-                       asc(g.c.contig),
-                       asc(g.c.position))
+
+        q = _add_ordering(q, g, 'String', 'contig', 'asc')
```

Comment:
`column_type` doesn't get used in this case, and the `contig` column certainly isn't sorted like a string. I'd just set it to something like `None` or `"n/a"` or `"(special)"` to avoid confusion.